src/types/TextColor.tsx

```diff
@@ -0,0 +1,3 @@
+export type TextColor = 'darkest' | 'dark' | 'light' | 'lightest'
+
+export const TextColorValues = ['darkest', 'dark', 'light', 'lightest']
```

Comment:
Let's change it like this also in other places where you've used same approach:

```suggestion
export const TextColor = ['darkest', 'dark', 'light', 'lightest'] as const

export type TextColor = typeof TextColor[number]
```